### DISCUSSION TITLE
Error: The connection is not allowed , The remote device is disabled.

### DISCUSSION THREAD

**Discussion by necklinux** [rustdesk/rustdesk-server-pro#415] [General]
Server: RustDesk Server Pro 1.4.4
RustDesk Client:
1. Windows 11 , RustDesk Version: 1.2.3-1
2. Android , lge-LM-G820 , Rustdesk Version: 1.3.1

I want to remote control android phone from windows 11 computer , get error message: The connection is not allowed , The remote device is disabled.
![error](https://github.com/user-attachments/assets/777887df-8fce-4b11-a637-f121f953e13c)

I log in RustDesk Server as Administrator, go to Device pages to enable this phone.
![4642](https://github.com/user-attachments/assets/2f2be58b-e80f-4b1d-a4b9-262bdcadc149)

But after I refresh this page . It shows that this phone disable again.
I have more than 40pcs Devices have same issue.
How can I enable these phones?
Thanks
Peter

**Comment by rustdesk** [comment]
You need to [upgrade your plan](https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/license/#renewupgrade-license) for more devices, or https://github.com/rustdesk/rustdesk-server-pro/discussions/182
